bin/run.js

```diff
@@ -27,6 +27,12 @@ try {
 }
 
 // process runfile.js
-console.log('Processing runfile...')
-var runfile = require(path.resolve('./runfile'));
+try {
+    console.log('Processing runfile...')
+    var runfile = require(path.resolve('./runfile'));
+} catch(e) {
```

Comment:
Actually, I was trying to follow your notations. I noticed that you had used `e` in the other try catch blocks. Anyways, I changed all the instances of `e` referring to error in try catch blocks, to `error`. :smile: